app/src/main/java/com/kickstarter/services/KSApolloClient.kt

```diff
@@ -520,6 +548,56 @@ class KSApolloClient(val service: ApolloClient) : ApolloClientType {
         }
     }
 }
+private fun backingObjectFromId(backingGr: GetBackingQuery.Backing?): Backing {
+
+    val name = backingGr?.backer()?.name() ?: ""
+    val id = decodeRelayId(backingGr?.id())?.let { it } ?: 0
+    val locationId = decodeRelayId(backingGr?.location()?.id())
+    val projectId = decodeRelayId(backingGr?.id()) ?: 0
+    val backerId = decodeRelayId(backingGr?.backer()?.id()) ?: 0
+    val rewardId = decodeRelayId(backingGr?.reward()?.id()) ?: 0
+    val avatar = Avatar.builder()
+            .medium(backingGr?.backer()?.imageUrl())
+            .build()
+    val rewardAmount = backingGr?.reward()?.amount()?.amount()?.toDouble()
+    val rewardSingleLocation = Reward.SingleLocation.builder()
+            .localizedName(backingGr?.location()?.displayableName())
+            .id(decodeRelayId(backingGr?.location()?.id())?: 0)
+            .build()
+
+    val reward = Reward.builder()
+            .minimum(rewardAmount?: 0.0)
+            .description(backingGr?.reward()?.description())
+            .estimatedDeliveryOn(DateTime(backingGr?.reward()?.estimatedDeliveryOn()))
+            .convertedMinimum(rewardAmount ?: 0.0)
```

Comment:
The `convertedMinimum` is based on the projects currency and exchange rate, we might need this to come through from GraphQL too.

Reply:
got it!